lib/wallet/src/Cardano/Wallet/DB/Store/Submissions/Layer.hs

```diff
@@ -1,35 +1,51 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
 {-# LANGUAGE GADTs #-}
+{-# LANGUAGE NoMonomorphismRestriction #-}
 {-# LANGUAGE OverloadedLabels #-}
+{-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE RecordWildCards #-}
 {-# LANGUAGE ViewPatterns #-}
+
 -- |
 -- Copyright: © 2022 IOHK
 -- License: Apache-2.0
 --
--- An implementation of the DBPendingTxs which uses Persistent and SQLite.
+-- Access to the submissions store.
 
 module Cardano.Wallet.DB.Store.Submissions.Layer
-    ( mkDbPendingTxs
-    , mkLocalTxSubmission)
+    ( mkLocalTxSubmission
+    , emptyTxSubmissions_
+    , resubmitTx_
+    , getInSubmissionTransactions_
+    , readLocalTxSubmissionPending_
+    , rollForwardTxSubmissions_
+    , removePendingOrExpiredTx_
+    , rollBackSubmissions_
+    , pruneByFinality_
+    , addTxSubmission_
+    , getInSubmissionTransaction_
```

Comment:
```suggestion
    , emptyTxSubmissions
    , resubmitTx
    , getInSubmissionTransactions
    , readLocalTxSubmissionPending
    , rollForwardTxSubmissions
    , removePendingOrExpiredTx
    , rollBackSubmissions
    , pruneByFinality
    , addTxSubmission
    , getInSubmissionTransaction
```

I think we can drop the underscores now and import these function qualified. I had added the underscore to e.g. `resubmitTx_` because it was a record field, now it has been downgraded to a "regular" function. 🤔